### DISCUSSION TITLE
Context menu add icons and keyboard shortcuts

### DISCUSSION THREAD

**Discussion by D8ma** [linuxdeepin/developer-center#3279] [Features Request & Ideas | 特性请求 & 头脑风暴]
![IMG_20220630_141113.jpg](https://user-images.githubusercontent.com/107427853/176663452-77accbfa-3630-4751-afd4-7dcf3ae877ee.jpg)![IMG_20220630_140905.jpg](https://user-images.githubusercontent.com/107427853/176663474-614b3841-0ba1-4dd1-9e63-c3bd34f98ca7.jpg)![IMG_20220630_135406.jpg](https://user-images.githubusercontent.com/107427853/176663495-e50e32aa-aebe-4023-92f8-f3ed4b444092.jpg)![IMG_20220630_141413.png](https://user-images.githubusercontent.com/107427853/176663845-dcdbcbda-17d0-4bb2-9b24-3f6a0393efd1.png)

**Comment by BLumia** [comment]
Thanks for creating these feature request issues, just to be clear, please use the [discussion board](https://github.com/linuxdeepin/developer-center/discussions) for feature request and ideas. I'll help you convert these issues to discussion thread tomorrow.

**Comment by BLumia** [comment]
Some additional info: currently not adding icons for these actions is by-design. The designer from deepin doesn't think adding them is a good idea. There is actually a (`dde-file-manager` scope) gsettings option to enable action icons for DDE but not enabled by default.

**Reply by D8ma** [reply]
Add a switch so we can turn this feature on, please.

**Comment by ghost** [comment]
I don’t think is a good idea, Deepin is not trying here to copy windows. And if you come from windows have to understand that Linux is fundamentally different

**Reply by BLumia** [reply]
I actually do think action icons and shortcut indicator string can be helpful for accessibility purposes. Other DE also have such support (KDE for example), but currently, not adding these icons and shortcut strings is a decision made by the designer :)

**Reply by raspin0** [reply]
Windows has nothing to do with this issue. I completely agree with the designer, the problem lies in how deepin context menu looks currently. A complete refactor of context menu is needed in order to introduce those accessibility features.
